channels_redis/pubsub.py

```diff
@@ -334,14 +332,14 @@ async def _get_pub_conn(self):
         if self._lock is None:
             self._lock = asyncio.Lock()
         async with self._lock:
-            if self._pub_conn is not None and self._pub_conn.closed:
-                self._put_redis_conn(self._pub_conn)
+            if self._pub_conn is not None and self._pub_conn.connection is None:
+                await self._put_redis_conn(self._pub_conn)
```

Comment:
It's not clear to me if this is still necessary now that we are using the built-in pool functionality from redis-py.

Reply:
If we are using the native pools functionality, this should not be necessary.

We should only be concerning ourselves with the creation and destruction of pools at that point, and redis-py can handle everything else.